webapp/src/features/dashboard/index.js

```diff
@@ -0,0 +1,9 @@
+import Dashboard from './Dashboard'
+import slice from './dashboard.slice'
+
+export const {
+	name,
+	actions: { setSelectedDashboardModal },
+	reducer,
+} = slice
+export default Dashboard
```

Comment:
@himanshuvicesoftware @aakashThapa0011 shouldn't the folder name be `Dashboard` instead of `dashboard` depending on what we are exporting by default ?

Reply:
Updated the folder name with PascalCase instead of camelCase.